src/slurm_plugin/instance_manager.py

```diff
@@ -848,7 +848,11 @@ def _best_effort_node_assignment(
             # No instances launched at all, e.g. CreateFleet API returns no EC2 instances,
             # or no left instances available from a best-effort EC2 launch
             logger.info("No launched instances found for nodes %s", print_with_count(nodes_resume_list))
-            self._update_failed_nodes(set(nodes_resume_list), "InsufficientInstanceCapacity", override=False)
+            self._update_failed_nodes(
+                set(nodes_resume_list),
+                "InsufficientInstanceCapacity(Check slurm_resume log for ec2 error codes)",
```

Comment:
[BLOCKING] I agree in making the log line more helpful, redirecting the user to the right log. However, doing it this way actually changes the error code from `InsufficientInstanceCapacity` to `InsufficientInstanceCapacity(Check...codes)`, which ultimately can have consequences in the way we monitor ICE errors on the clister dashboard.
See https://github.com/aws/aws-parallelcluster-node/blob/1b4ba77d5d4cb0990868c63e84cc607e22669d16/src/slurm_plugin/cluster_event_publisher.py#L27